usb/util.py

```diff
@@ -254,22 +302,15 @@ def get_string(dev, index, langid = None):
     The return value is the unicode string present in the descriptor.
     """
     from usb.control import get_descriptor
+    langids = dev.langids
+
+    if 0 == len(langids):
+        raise ValueError("get_string from a device that offers none")
```

Comment:
All think this can be removed. If an app wants to make sure the langid exists, if can do it by itself.